src/main/java/org/opentripplanner/standalone/OTPMain.java

```diff
@@ -118,7 +119,7 @@ public void run() {
                 }
             } else {
                 LOG.error("An error occurred while building the graph. Exiting.");
```

Comment:
This log message is no longer accurate. OTP may not exit if this method was not called from main(). Please remove the "exiting" part.

Reply:
Fixed